sdk/tables/data-tables/samples/typescript/src/batchOperations.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { TableClient } from "../../..";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const connectionString = process.env["ACCOUNT_CONNECTION_STRING"] || "";
+
+export async function batchOperations() {
+  console.log("== Batch Operations Sample ==");
+
+  // Note that this sample assumes that a table with tableName exists
+  const tableName = "batch";
+
+  // See authenticationMethods sample for other options of creating a new client
+  const client = TableClient.fromConnectionString(connectionString, tableName);
+
+  // await client.create();
+
+  const partitionKey = "Stationery";
+
+  const entities: Entity[] = [
+    {
+      partitionKey,
+      rowKey: "A1",
+      name: "Marker Set",
+      price: 5.0,
+      quantity: 21
+    },
+    {
+      partitionKey,
+      rowKey: "A2",
+      name: "Pen Set",
+      price: 2.0,
+      quantity: 6
+    },
+    {
+      partitionKey,
+      rowKey: "A3",
+      name: "Pencil",
+      price: 1.5,
+      quantity: 100
+    }
+  ];
+
+  // Create the new batch
+  const batch = await client.createBatch(partitionKey);
+
+  // Add a createEntities operation to the batch
+  await batch.createEntities(entities);
+
+  // Submit the batch
+  const response = await batch.submitBatch();
+
+  console.log(response);
+
+  // await client.delete();
```

Comment:
is this not needed?

Reply:
Similarly, the commented out create call on L21 is also not needed (as the comment above says it expects the table to exist).  I suspect this was here to make it easier to run the sample over and over during development and get back to a clean state each time.